cmd/controller-manager/main.go

```diff
@@ -52,13 +52,21 @@ import (
 )
 
 var featureGates map[string]bool
+var fedNamespace string
+var clusterNamespace string
+var limitedScope bool
 var installCRDs = flag.Bool("install-crds", true, "install the CRDs used by the controller as part of startup")
 
 // Controller-manager main.
 func main() {
 	verFlag := flag.Bool("version", false, "Prints the version info of controller-manager")
 	flag.Var(flagutil.NewMapStringBool(&featureGates), "feature-gates", "A set of key=value pairs that describe feature gates for alpha/experimental features. "+
 		"Options are:\n"+strings.Join(utilfeature.DefaultFeatureGate.KnownFeatures(), "\n"))
+	flag.StringVar(&fedNamespace, "federation-namespace", util.DefaultFederationSystemNamespace,
+		fmt.Sprintf("The namespace the federation control plane is deployed in.  If unset, will default to %q.", util.DefaultFederationSystemNamespace))
+	flag.StringVar(&clusterNamespace, "cluster-namespace", util.MulticlusterPublicNamespace,
```

Comment:
I think here should be `registry-namespace` based on this PR https://github.com/kubernetes-sigs/federation-v2/pull/225